src/iexamine.cpp

```diff
@@ -1555,6 +1555,40 @@ void iexamine::flower_poppy( player &p, const tripoint &examp )
     }
 }
 
+/**
+ * Prompt pick (or drink nectar if able) poppy bud. Not safe for player.
+ *
+ * Drinking causes: -25 hunger, +20 fatigue, pkill2-70 effect and, 1 in 20 pkiller-1 addiction.
+ * Picking w/ env_resist < 5 causes 1 in 3  sleep for 12 min and 4 dmg to each leg
+ */
```

Comment:
Change comments to reflect the actual info on cactus pads, or remove them altogether.